src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/sqs/domainevent/AdditionalInformation.kt

```diff
@@ -16,4 +16,7 @@ data class AdditionalInformation(
 
   @JsonProperty("targetCRN")
   val targetCrn: String? = null,
+
+  @JsonProperty("removedNomsNumber")
+  val targetPrisonNumber: String? = null,
```

Comment:
the removed one is technically the 'source'. As the removed gets merged into the nomisNumber